# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-images/tiled-gradients.html.ini
# Commit: f733b37f9626
# Full Hash: f733b37f9626548ae83564a3fd43ddfa972977d9
# Author: Nicolas Silva <nsilva@mozilla.com>
# Date: 2021-04-22 21:31:57
# Regressor Bug: 1702228
# File Overlap Count: 1
# Description:
#   Bug 1702228 - Adjust reftests. r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112970
# ==============================================================================

diff -r 0919633e25e9 -r f733b37f9626 testing/web-platform/meta/css/css-images/tiled-gradients.html.ini
--- a/testing/web-platform/meta/css/css-images/tiled-gradients.html.ini	Thu Apr 22 10:34:55 2021 +0000
+++ b/testing/web-platform/meta/css/css-images/tiled-gradients.html.ini	Thu Apr 22 10:34:55 2021 +0000
@@ -1,5 +1,3 @@
 [tiled-gradients.html]
   expected:
-    if webrender: FAIL
     if swgl and os == "mac": FAIL
-  bug: 1646043